io/plugins/eu.esdihumboldt.hale.io.haleconnect.ui/src/eu/esdihumboldt/hale/io/haleconnect/ui/projects/ChooseHaleConnectProjectWizardPage.groovy

```diff
@@ -73,7 +73,7 @@ import groovy.transform.CompileStatic
 public class ChooseHaleConnectProjectWizardPage extends ConfigurationWizardPage<HaleConnectProjectConfig> {
 
 	private static final ALogger log = ALoggerFactory.getLogger(ChooseHaleConnectProjectWizardPage.class);
-	private static final OwnerFilterEntry NULL_FILTER = new OwnerFilterEntry(owner: null as Owner[], label: "All projects");
+	private static final OwnerFilterEntry NULL_FILTER = new OwnerFilterEntry(owner: Arrays.asList(null) , label: "All projects");
```

Comment:
I think this is different than the original. The original used `null` for the array itself, you create a list here with a `null` inside.

Reply:
True, updated the code.